### ISSUE TITLE
Error in "CNTK_301_Image_Recognition_with_Deep_Transfer_Learning"

### ISSUE THREAD

**Issue by raushankumar-iitkgp** [Microsoft/CNTK#1942]
I'm running CNTK_301_Image_Recognition_with_Deep_Transfer_Learning ipython from tutorial directory in jupyter notebook on amazon aws Deep Learning AMI Ubuntu Linux - 1.3_Apr2017 server.
It shows 
" ValueError: all the input array dimensions except for the concatenation axis must match exactly "
 in line
"  class_probs = np.column_stack((probs, animals_data['class_mapping'])).tolist() "
in # evaluate test images 
The error shows like this ...
ValueError                                Traceback (most recent call last)
<ipython-input-19-6c79f9dec64c> in <module>()
      9         if probs[0]=='None':
     10             continue
---> 11         class_probs = np.column_stack((probs, animals_data['class_mapping'])).tolist()
     12         class_probs.sort(key=lambda x: float(x[0]), reverse=True)
     13         predictions = ' '.join(['%s:%.3f' % (class_probs[i][1], float(class_probs[i][0]))                                 for i in range(0, animals_model['num_classes'])])

/usr/local/lib/python3.4/dist-packages/numpy/lib/shape_base.py in column_stack(tup)
    319             arr = array(arr, copy=False, subok=True, ndmin=2).T
    320         arrays.append(arr)
--> 321     return _nx.concatenate(arrays, 1)
    322 
    323 def dstack(tup):

ValueError: all the input array dimensions except for the concatenation axis must match exactly

**Comment by raushankumar-iitkgp** [comment]
@cha-zhang please look at this matter

**Comment by sayanpa** [comment]
Hi Raushankumar,
I just ran the code on my local machine with the latest released binaries. I print the following:
```
        print(probs)
        print(animals_data['class_mapping'])
        print(class_probs)
```
The result i get are:
```
    [ 0.93803453  0.06196541]
    ['Sheep' 'Wolf']
    [['0.938034534454', 'Sheep'], ['0.0619654096663', 'Wolf']]
```
I have tried it in py2.7 and py3.5. Can you try writing a small test code. Give the first two lines as input and see if you can generate the 3rd line.

Thanks,
Sayan

**Comment by raushankumar-iitkgp** [comment]
@sayanpa thanks for responding.
print CNTK.version didn't work for me. I used pip list and found cntk (2.0rc1) , hope this the thing you asked for.
and for those input I can't feed them as it shows syntax error.
![03](https://cloud.githubusercontent.com/assets/16998703/26773628/b68d9154-49e9-11e7-83a4-ca911cbf8d1a.png)
in addition to this, I used to get these 2 warnings
![01](https://cloud.githubusercontent.com/assets/16998703/26773644/ca6c8b9e-49e9-11e7-8157-57521734d754.png)
 
![02](https://cloud.githubusercontent.com/assets/16998703/26773653/d32b41e4-49e9-11e7-944f-6995d4f8a7a0.png)

 I'm running the code in jupyter notebook on amazon aws Deep Learning AMI Ubuntu Linux - 1.3_Apr2017 server.
